components/Dashboard/FormPane.tsx

```diff
@@ -140,7 +151,11 @@ const FormPane: React.FC = () => {
                   {form?.name}
                 </span>
                 <span className="flex items-center gap-1 truncate text-sm text-gray-500">
-                  0 responses. Created{' '}
+                  <Link href="/dashboard/web/testimonials" className="underline">
```

Comment:
Handle the check for 0 responses also, 
if the value of responses is 0 then show the `0 responses` text else the `<Link>` component